docs/security-monitoring.md

```diff
@@ -1,29 +1,29 @@
 # Security Monitoring
 
-Private Packagist Security Monitoring searches the dependencies of your private packages for known security vulnerabilities.
-Packages are analyzed every time we find a new commit in your package and when a new vulnerability is published in one of the databases.
+Private Packagist Security Monitoring searches the dependencies of your projects (private packages with a composer.lock) for known security vulnerabilities.
+Projects are analyzed every time we find a new commit in your project and when a new vulnerability is published in one of the databases.
 
-The following databases are used to analyze your packages:
+The following databases are used to analyze your projects:
 * [FriendsOfPHP/security-advisories](https://github.com/FriendsOfPHP/security-advisories)
 
 <div class="row column">
     <div class="callout warning">
-        <p>Security monitoring is available for packages in your organization. To monitor your projects, you must add them to Private Packagist as packages, even if you do not intend to install them as a dependency.</p>
+        <p>Security monitoring is available for projects in your organization. To monitor your projects, you must add them to Private Packagist as packages, even if you do not intend to install them as a dependency.</p>
     </div>
 </div>
 
 
 ### Configuring Monitoring Settings
-The default branch of every private package is automatically monitored if a composer.lock file is present.
+The default branch of every project is automatically monitored if a composer.lock file is present.
```

Comment:
```suggestion
The default branch of every project is automatically monitored if a composer.lock file is present.
```

Reply:
@naderman there are not suggested changes here. Did you miss something?

Reply:
haha no i meant to undo this, and didn't realize that it's still a "comment" so I cannot just edit the content to be the original to undo the comment :D